File: 2-Fundamentals-Java/BasicSyntax_ConditionalStatementsAndLoops/Exercises/PrintAndSum.java
package BasicSyntax_ConditionalStatements_Loops.Exercises;

import java.util.Scanner;

public class PrintAndSum {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        int firstNumber = Integer.parseInt(scanner.nextLine());
        int endNumber = Integer.parseInt(scanner.nextLine());
        int sum = 0;

        for (int i = firstNumber; i <= endNumber; i++) {
            System.out.print(i + " ");
            sum += i;

        }
        System.out.println();
        System.out.println("Sum: " + sum);
    }
}

File: 6-JS Front-End/SyntaxFundamentals/Exercises/PrintAndSum.js
function sumInterval(firstNumber, secondNumber) {
    let sum = 0;
    let allNumbers = '';
    for(let i = firstNumber; i <= secondNumber; i++) {
        sum += i;
        allNumbers += i + ' ';
    }

    console.log(allNumbers);
    console.log(`Sum: ${sum}`);
}

sumInterval(5, 10);
sumInterval(0, 26);
sumInterval(50, 60);